docs/concepts/configuration/manage-compute-resources-container.md

```diff
@@ -135,8 +135,8 @@ When using Docker:
   [`--cpu-shares`](https://docs.docker.com/engine/reference/run/#/cpu-share-constraint)
   flag in the `docker run` command.
 
-- The `spec.containers[].resources.limits.cpu` is converted to its millicore value,
-  multiplied by 100000, and then divided by 1000. This number is used as the value
+- The `spec.containers[].resources.limits.cpu` is converted to its millicore value, and
```

Comment:
Thanks @vishh. Do you mean the entire description need a revisit ?

`This number is used as the value
  of the [`--cpu-quota`](https://docs.docker.com/engine/reference/run/#/cpu-quota-constraint)
  flag in the `docker run` command.  The [`--cpu-period`] flag is set to 100000,
   which represents the default 100ms period for measuring quota usage. The
   kubelet enforces cpu limits if it is started with the
  [`--cpu-cfs-quota`] flag set to true. As of Kubernetes version 1.2, this flag
  defaults to true.`

What I understand is that the default cpu quota can be changed to this number and is set in the `--cpu-quota` flag of the docker command.
Please correct me.